core/invoker/src/main/scala/whisk/core/container/HttpUtils.scala

```diff
@@ -85,6 +87,9 @@ protected[container] class HttpUtils(
             val containerResponse = Option(response.getEntity).map { entity =>
                 val statusCode = response.getStatusLine.getStatusCode
                 val contentLength = entity.getContentLength
+                if(statusCode == 503 && isProxied) {
```

Comment:
is this necessary? I had understood the issue was related to the unit tests for the container runtimes - not the invoker to container communication. If so, then these changes are not necessary since this class is not used for those unit tests.